docs/light/load_balancer.md

```diff
@@ -1,4 +1,4 @@
-# Load Balancing Module - WIP
+# Load Balancing
```

Comment:
This file shouldn't be included. It's mostly a spec

Reply:
Pulling